### DISCUSSION TITLE
Create a ticket and add users in copy (email)

### DISCUSSION THREAD

**Discussion by developersheat** [mozts2005/ZendeskApi_v2#581] [Q&A] [answered]
Hi I am creating a Ticket and I don't see how to add users in copy to the ticket
Ex: I want to add the user test.gmail.com to get notified that a new ticket was created so he can check the ticket

**Comment by mozts2005** [comment]
I think what you want are CCs or followers. If enabled on your account. For more details see https://support.zendesk.com/hc/en-us/articles/4408822451482

**Reply by mozts2005** [reply, marked as answer]
Did a quick check and it looks like the required properties for using the cc and follower feature are missing. Feel free to submit a pull request to add support. If you open an issue I will try and add support soon but if you need it faster a pr is your best option.
